compat/compat_mman.c

```diff
@@ -109,174 +144,128 @@ DWORD map_prot_flags(int prot, DWORD *dwDesiredAccess)
 	return 0;
 }
 
-void *mmap(void *start, size_t length, int prot, int flags, int fd,
+void *mmap(void *addr, size_t length, int prot, int flags, int fd,
```

Comment:
Namespace this compatibility wrapper to `bt_mmap()`.

The function should also be `BT_HIDDEN`.